detekt-config.yml

```diff
@@ -1,5 +1,5 @@
 build:
-  maxIssues: 4
+  maxIssues: 5
```

Comment:
Where is the problem? If the service has too many functions, add suppress annotation.